Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -134,12 +138,58 @@ private void AutoSuggestBox_SuggestionChosen(AutoSuggestBox sender, AutoSuggestB
 
         private void AutoSuggestBox_TextChanged(AutoSuggestBox sender, AutoSuggestBoxTextChangedEventArgs args)
         {
-            QueryTextChanged?.Invoke(sender, args);
+            string t = sender.Text.Trim();
+
+            if (t.Contains(TokenDelimiter))
+            {
+                bool lastDelimited = t[t.Length - 1] == TokenDelimiter[0];
+
+                string[] tokens = t.Split(TokenDelimiter);
+                int numberToProcess = lastDelimited ? tokens.Length : tokens.Length - 1;
+                for (int position = 0; position < numberToProcess; position++)
+                {
+                    string token = tokens[position];
+                    token = token.Trim();
+                    if (token.Length > 0)
+                    {
+                        AddToken(token);
+                    }
+                }
+
+                if (lastDelimited)
+                {
+                    sender.Text = string.Empty;
+                }
+                else
+                {
+                    sender.Text = tokens[tokens.Length - 1];                }
```

Comment:
```suggestion
                    sender.Text = tokens[tokens.Length - 1];                
                }
```

Looks like we missed one somehow, I'll see if I can merge it in and we should be good.
```
       "D:\a\1\s\Windows Community Toolkit.sln" (Build target) (1) ->
       "D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj" (default target) (9) ->
       (CoreCompile target) -> 
         TokenizingTextBox\TokenizingTextBox.cs(165,77): error SA1500: Braces for multi-line statements must not share line [D:\a\1\s\Microsoft.Toolkit.Uwp.UI.Controls\Microsoft.Toolkit.Uwp.UI.Controls.csproj]

    0 Warning(s)
    1 Error(s)
```

I've also updated the branch from master so it'll be easier to merge.